kratos/utilities/mortar_utilities.h

```diff
@@ -384,7 +384,7 @@ class MortarUtilities
         
         const double this_norm = norm_2(normal);
         
-        KRATOS_DEBUG_ERROR_IF(this_norm < ZeroTolerance) << "Zero norm normal vector. Norm:" << this_norm << std::endl;
+        KRATOS_DEBUG_ERROR_IF(this_norm < std::numeric_limits<double>::epsilon()) << "Zero norm normal vector. Norm:" << this_norm << std::endl;
```

Comment:
accepting but...why don'y you use "zero_tolerance" here too?

Reply:
No, because is used just once